src/get-started/quickstart.md

```diff
@@ -74,8 +88,9 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
 ```
 
 {:.runner-flink}
-``` 
-TODO BEAM-899
+```
+$ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
```

Comment:
Could we teach the users here how to run on Flink locally and on a remote cluster?